ldpd/ldpd.c

```diff
@@ -500,8 +500,11 @@ start_child(enum ldpd_process p, char *argv0, int fd_async, int fd_sync,
 		argv[argc++] = (char *)ctl_sock_custom_path;
 	}
 	/* zclient serv path */
+#ifdef HAVE_TCP_ZEBRA
```

Comment:
Since the other file use the same pattern "ifdef ... else ...." and I want to keep the coding style consistency. 
If we have to add third definition about zcliet connection in the future, I think the pattern consistency can avoid us rearranging the code.